Automattic/jetpack
# Reorg: Enable symlinks within Docker

The major problem with our Docker setup previously post-reorg was that package symlinks inside and outside docker didn't match up.

The main thing this does is fix that so it all just works.

package.json between repo root and the jetpack plugin root... which docker commands should live where? Added a card for that.

#### Changes proposed in this Pull Request:
* Reorg: Make docker work.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* FROM REPO ROOT, `yarn docker:clean && yarn docker:build-image` (to make sure you're starting fresh), `yarn docker:up`, `yarn docker:install`
* Visit localhost/wp-admin and confirm Jetpack is present and active already. Jetpack Debug Helper should be there too.
* For bonus points, `yarn docker:down` then mkdir a new plugin within `projects/plugins`, then `yarn docker:up`. That should symlink the new folder. You can then `yarn docker:sh` and `ls -al wp-content/plugins` to see new folder linked.
![2020-12-18 at 1 55 PM](https://user-images.githubusercontent.com/88897/102655912-ad83c480-4138-11eb-8f07-89072396ed24.png)


#### Proposed changelog entry for your changes:
* n/a
